@types/environment.d.ts

```diff
@@ -0,0 +1,11 @@
+declare global {
+  namespace NodeJS {
+    interface ProcessEnv {
+      readonly NODE_ENV: 'development' | 'production' | 'test' | 'build';
+    }
+  }
+}
+
+// If this file has no import/export statements (i.e. is a script)
+// convert it into a module by adding an empty export statement.
```

Comment:
nit: remove this comment

Reply:
Okay, taking those out now

Reply:
I've actually taken off this environment type override entirely due to the type conflicts I mentioned.